### PR TITLE
add eliuds-eggs exercise

### PR DESCRIPTION
**Pull request by BNAndras**
Related to #675

### CODE REVIEW

**Comment by BNAndras** [comment]
@exercism/Julia, this is also a #48in24 exercise, but it's not featured until week 43.

**Review by cmcaine — APPROVED** [review]
Thanks, seems fine! Would be nice to have an approaches file, but not necessary.

**Comment by BNAndras** [comment]
Sure. Approaches would probably be something like recursion, while loops, and then the built-in I use.

File: exercises/practice/eliuds-eggs/runtests.jl [hunk 1]
```diff
@@ -0,0 +1,21 @@
+using Test
+
+include("eliuds-eggs.jl")
+
+@testset verbose = true "Eliud's Eggs" begin
```
Comment (ErikSchierboom) [inline exercises/practice/eliuds-eggs/runtests.jl:5]:
```suggestion
@testset verbose = true "tests" begin
```

**Pull request merged**